### ISSUE TITLE
Senden des Pakets mit integrierten Titeldaten

### ISSUE THREAD

**Issue by philboeselager** [hbz/laser-ygor#304]
Für den automatischen Update-Prozess und den "all in one go" Prozess, der durch das neue GOKb-UI angestoßen wird, ist der integrierte Prozess "Senden des Pakets inklusive Titeldaten" bereits entwickelt worden. Dieses Ticket betrifft die Einrichtung dieser Sende-Option als Button neben den bereits bestehenden Buttons "Titel Senden" und "Paket Senden".

@MiriKon @darupp : Ist das sinnvoll?

**Comment by darupp** [comment]
Ja, das ist auf jeden Fall sinnvoll. Ich würde den Button als ersten setzen und farblich positiv hervorheben, da das der Standardfall werden dürfte.

**Comment by philboeselager** [comment]
@darupp Ich füge ebenfalls einen Button für den Export der mit Titeln integrierten Paket-Daten hinzu, ja?

**Comment by darupp** [comment]
Ja, das ist sinnvoll!
